tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -340,7 +342,7 @@ IncludeByKey.PASSTHROUGH {
 						out_ch_map	$CHANNEL_MAP_3_POINT_1
 					}
 					Object.Control.mixer.1 {
-						name 'Main Capture Volume 3'
+						name 'Pre Mux $DMIC0_PCM_0_NAME Capture Volume'
```

Comment:
again where is there a MUX on DMIC capture?

Reply:
that's demux...

Reply:
... Ok, will fix.